src/lavinmq/http/controller/parameters.cr

```diff
@@ -73,6 +73,9 @@ module LavinMQ
             unless value
               bad_request(context, "Field 'value' is required")
             end
+            if name.size > 255
+              bad_request(context, "#{component} name too long, can't exceed 255 characters")
```

Comment:
are you sure about parameters not supporing longer names? afaik they are never transfered over the amqp protocol and should not have a limitation, but maybe they are in rabbit?

Reply:
No, you're right. Only had issues with long shovel names because we used name as consumer tag as we fixed in another PR.